Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -190,6 +192,19 @@ readNoPreambleDicom(std::ifstream & file) // NOTE: This file is duplicated in it
   return true;
 }
 
+static void convert_YCbCr(unsigned char * p, const size_t len, const uint8_t t)
+{
+  if (len % 3 != 0) return;
+  for (size_t x = 0; x < len; x += 3)
+  {
+    const unsigned char in[] = { p[x], p[x + 1], p[x + 2] };
+    if (t == 1)
```

Comment:
The `t` parameter should be `bool`, because that's how it is treated. The name `t` is also quite non-descriptive, it could be called something like `const bool ybr2rgb`. Consequently, `if (t == 1)` then turns into `if (ybr2rgb)`